### PR TITLE
[17.0][MIG] partner_property: Migration to version 17.0

### PR DESCRIPTION
**Pull request by pilarvargas-tecnativa**
cc @Tecnativa TT51236

@victoralmau @chienandalu please review

### CODE REVIEW

File: partner_property/readme/USAGE.md [hunk 1]
```diff
@@ -1,4 +1,12 @@
+Delete property fields:
```
Comment (chienandalu) [inline partner_property/readme/USAGE.md]:
```suggestion
Add property fields:

**Review by chienandalu — COMMENTED** [review]
Test and code :+1: Just a tiny change:

File: partner_property/readme/USAGE.md (hunk 1 above)
Comment (pilarvargas-tecnativa) [reply partner_property/readme/USAGE.md]: thanks!

File: partner_property/i18n/es.po [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+#: model:ir.model.fields,field_description:partner_property.field_res_partner__properties_type_person
+#: model:ir.model.fields,field_description:partner_property.field_res_users__properties_type_person
+msgid "Properties Type Person"
+msgstr ""
+
+#~ msgid "Partner Properties"
+#~ msgstr "Propiedades de contacto"
+
+#~ msgid "Properties"
+#~ msgstr "Propiedades"
```
Comment (victoralmau) [inline partner_property/i18n/es.po]:
```suggestion
```

File: partner_property/i18n/it.po [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+#: model:ir.model.fields,field_description:partner_property.field_res_partner__properties_type_person
+#: model:ir.model.fields,field_description:partner_property.field_res_users__properties_type_person
+msgid "Properties Type Person"
+msgstr "Persona tipo proprietà"
+
+#~ msgid "Partner Properties"
+#~ msgstr "Prorietà partner"
+
+#~ msgid "Properties"
+#~ msgstr "Proprietà"
```
Comment (victoralmau) [inline partner_property/i18n/it.po]:
```suggestion
```

File: partner_property/views/res_partner_views.xml [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+        <field name="name">res.partner.form</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="base.view_partner_form" />
+        <field name="arch" type="xml">
+            <notebook position="before">
+                <field name="properties_company_id" invisible="1" />
+                <field
+                    name="properties_type_company"
+                    columns="2"
+                    invisible="is_company == False"
```
Comment (victoralmau) [inline partner_property/views/res_partner_views.xml]:
```suggestion
                    invisible="not is_company"
```

File: partner_property/views/res_partner_views.xml [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+                />
+                <field
+                    name="properties_type_person"
+                    columns="2"
+                    invisible="is_company == True"
```
Comment (victoralmau) [inline partner_property/views/res_partner_views.xml]:
```suggestion
                    invisible="is_company"
```

**Review by victoralmau — CHANGES_REQUESTED** [review]
Functional review OK. Can you squash administrative commits?

**Comment by pilarvargas-tecnativa** [comment]
@victoralmau all changes done, thanks!

**Review by victoralmau — APPROVED** [review]

**Review by chienandalu — APPROVED** [review]

**Pull request merged**